src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -348,9 +351,18 @@ def add_organization_member(
                 )
         return serialize_member(org_member)
 
-    def add_team_member(self, *, team_id: int, organization_member: RpcOrganizationMember) -> None:
+    def check_for_single_team(self, *, organization_id: int) -> Optional[RpcTeam]:
```

Comment:
The naming on this confused me a bit at first - I expected it to return a bool due to the "check". Not sure I have any better suggestions though, maybe `get_single_team`?